test/k8s-integration/.dockerized-kube-version-defs

```diff
@@ -0,0 +1,5 @@
+KUBE_GIT_COMMIT='somefakecommit'
+KUBE_GIT_TREE_STATE='dirty'
+KUBE_GIT_VERSION='v1.13.0-fake-testing-master.version'
+KUBE_GIT_MAJOR='1'
+KUBE_GIT_MINOR='13+'
```

Comment:
is it actually 1.13, or just whatever release is head?

Reply:
whatever release is head, but this is just a dummy file I reverse engineered so that build doesn't complain. I can make it something more obviously "dummy value" like `0` if you'd prefer

Reply:
yeah make it something like 999